### ISSUE TITLE
http://vlarena.opengvlab.com is inaccessible now...

### ISSUE THREAD

**Issue by zhimin-z** [OpenGVLab/Multi-Modality-Arena#27]
<img width="859" alt="image" src="https://github.com/user-attachments/assets/a9583576-4501-47ca-ae50-7fd2fce7878b">
This happens whichever network I have even attempted...However, I indeed could access to the arena or leaderboard a few months ago...

**Comment by manglu097** [comment]
Yep, demo can not be accessed normall right now. Please solve this problem, thanks.

**Comment by zhimin-z** [comment]
> Yep, demo can not be accessed normall right now. Please solve this problem, thanks.

It seems completely offline now:

![Image](https://github.com/user-attachments/assets/82adc861-15b9-4c24-badc-8f55579cb918)
